components/services/services.tsx

```diff
@@ -0,0 +1,29 @@
+import { IServices } from "../../fixtures/services.fixture";
+import styles from "./srvices.module.scss";
+ 
+const Services = ({ title="Services" }: IServices) => {
+  return (
+    <div className={styles.services}>
+      <div className={styles["services-wrapper"]}>
+        <div className={styles["services-title-wrapper"]}>
+          <h2 className={styles["services-title"]}>
+            <span className={styles["services-title-span"]}>\\ {title}</span>
+          </h2>
+        </div>
+        <div className={styles["services-text-wrapper"]}>
+          <p className={styles["services-text"]}>
+            We listen to your current needs, pain points during operations
```

Comment:
Move all text into the fixtures and simply import from there. This way your component will still be dynamic